src/plugins/mutes.js

```diff
@@ -18,5 +18,18 @@ export default function mutesPlugin () {
       mute,
       unmute
     })
+
+    function onModMute (ref) {
+      mp.emit('modMute', {
+        moderator: ref.m,
```

Comment:
Hmm, you're right. I was going off of this: https://github.com/plugcommunity/documentation/blob/master/api/events/backend_events/modMute.md